### DISCUSSION TITLE
How to cycle through search results

### DISCUSSION THREAD

**Discussion by cyberksh** [ibhagwan/fzf-lua#1591] [Q&A] [answered]
Using the new lazyvim 14.0 with the default config. I use `ctrl-p` and `ctrl-n` to go to the previous and next file search. How do I make fzf-lua go to the last search when I hit `ctrl-p` at the top of the search?

**Comment by ibhagwan** [comment, marked as answer]
#1202

**Reply by cyberksh** [reply]
sorry thanks! I should have used lesser keywords in the search.
